shared-libs/rules-engine/src/refresh-rules-emissions.js

```diff
@@ -11,39 +11,54 @@ const rulesEmitter = require('./rules-emitter');
 const TaskStates = require('./task-states');
 const transformTaskEmissionToDoc = require('./transform-task-emission-to-doc');
 
+const DEFAULT_OPTIONS = { enableTasks: true, enableTargets: true };
+
 /**
  * @param {Object[]} freshData.contactDocs A set of contact documents
  * @param {Object[]} freshData.reportDocs All of the contacts' reports
  * @param {Object[]} freshData.taskDocs All of the contacts' task documents (must be linked by requester to a contact)
  * @param {Object[]} freshData.userContactId The id of the user's contact document
+ * 
  * @param {int} calculationTimestamp Timestamp for the round of rules calculations
+ * 
+ * @param {Object=} [options] Options for the behavior when refreshing rules
+ * @param {Boolean} [options.enableTasks=true] Flag to enable tasks
+ * @param {Boolean} [options.enableTargets=true] Flag to enable targets
  *
- * @returns {Object[]} Array of updated task documents
+ * @returns {Object} result
+ * @returns {Object[]} result.targetEmissions Array of raw target emissions
+ * @returns {Object[]} result.updatedTaskDocs Array of updated task documents
  */
-module.exports = (freshData = {}, calculationTimestamp = Date.now()) => calculateFreshTaskDocs(freshData, calculationTimestamp)
-  .then(freshResults => {
-    const freshTaskDocs = freshResults.map(doc => doc.taskDoc);
-    const cancelledDocs = getCancellationUpdates(freshTaskDocs, freshData.taskDocs, calculationTimestamp);
-    const updatedTaskDocs = freshResults.filter(doc => doc.isUpdated).map(result => result.taskDoc);
-    return [...updatedTaskDocs, ...cancelledDocs];
-  });
+module.exports = (freshData = {}, calculationTimestamp = Date.now(), options = DEFAULT_OPTIONS) => {
```

Comment:
Inline the default options and destructure properly, eg: `{ enableTasks=true, enableTargets=true }={}`.